spec/unit/installer/xcode/pods_project_generator/target_installer_spec.rb

```diff
@@ -54,7 +54,7 @@ class PodsProjectGenerator
             @pod_target.stubs(:uses_swift?).returns(true)
             @installer.send(:add_target)
             @installer.send(:native_target).resolved_build_setting('SWIFT_OPTIMIZATION_LEVEL').should == {
-              'Release' => nil,
+              'Release' => '-Owholemodule',
```

Comment:
Yes, this is the default optimization build setting (for release) since Xcode 8.0.
https://swift.org/blog/whole-module-optimizations/